.changeset/calm-bottles-laugh.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/react': major
```

Comment:
This is not a major change, yeah?

Reply:
no. This is not a breaking change but it could affect typings downstream.

I wanna say this is a patch since it doesn't break anything.